packages/email/processingEmails/submitEmails.ts

```diff
@@ -62,4 +63,5 @@ export const handler = middy(baseHandler)
   .use(jsonBodyParser())
   .use(validator({ inputSchema: schemaWrapper }))
   .use(JSONErrorHandlerMiddleware())
-  .use(cors({ headers: "*", methods: "*" }));
+  .use(cors({ headers: "*", methods: "*" }))
+  .use(IpCheckerMiddleware());
```

Comment:
I think this one was missed by accident since you made the `IpCheckerMiddleware` first on the other ones.